### PR TITLE
fix incorrect links in github linting comment

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

**Review by mirpedrol — APPROVED** [review]
LGTM

**Comment by mirpedrol** [comment]
Docs for v2.10 should exist, maybe there was a problem building them during the release, although everything looks good from the [GitHub action](https://github.com/nf-core/tools/actions/runs/6300992423/job/17105031115).

**Comment by kedhammar** [comment]
I tried testing this by:
1. Opening a virtual codespace
2. Doing `pip install -e .` to deploy
3. Creating a new test pipeline using `nf-core create`
4. Vandalizing the assets/multiqc_config.yml
5. Running `nf-core lint` --> Works as expected
6. Running `nf-core lint --markdown lint.md` --> Crashes, gives the following error

````
INFO     Writing lint results to lint.md                                                                                                                                                                                                                                                                    
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /opt/conda/bin/nf-core:8 in <module>                                                             │
│                                                                                                  │
│   5 from nf_core.__main__ import run_nf_core                                                     │
│   6 if __name__ == '__main__':                                                                   │
│   7 │   sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])                         │
│ ❱ 8 │   sys.exit(run_nf_core())                                                                  │
│   9                                                                                              │
│                                                                                                  │
│ /workspaces/tools/nf_core/__main__.py:111 in run_nf_core                                         │
│                                                                                                  │
│    108 │   │   │   log.debug(f"Could not check latest version: {e}")                             │
│    109 │   │   stderr.print("\n")                                                                │
│    110 │   # Launch the click cli                                                                │
│ ❱  111 │   nf_core_cli(auto_envvar_prefix="NFCORE")                                              │
│    112                                                                                           │
│    113                                                                                           │
│    114 @click.group(context_settings=dict(help_option_names=["-h", "--help"]))                   │
│                                                                                                  │
│ /opt/conda/lib/python3.11/site-packages/click/core.py:1157 in __call__                           │
│                                                                                                  │
│   1154 │                                                                                         │
│   1155 │   def __call__(self, *args: t.Any, **kwargs: t.Any) -> t.Any:                           │
│   1156 │   │   """Alias for :meth:`main`."""                                                     │
│ ❱ 1157 │   │   return self.main(*args, **kwargs)                                                 │
│   1158                                                                                           │
│   1159                                                                                           │
│   1160 class Command(BaseCommand):                                                               │
│                                                                                                  │
│ /opt/conda/lib/python3.11/site-packages/rich_click/rich_group.py:21 in main                      │
│                                                                                                  │
│   18 │                                                                                           │
│   19 │   def main(self, *args, standalone_mode: bool = True, **kwargs):                          │
│   20 │   │   try:                                                                                │
│ ❱ 21 │   │   │   rv = super().main(*args, standalone_mode=False, **kwargs)                       │
│   22 │   │   │   if not standalone_mode:                                                         │
│   23 │   │   │   │   return rv                                                                   │
│   24 │   │   except click.ClickException as e:                                                   │
│                                                                                                  │
│ /opt/conda/lib/python3.11/site-packages/click/core.py:1078 in main                               │
│                                                                                                  │
│   1075 │   │   try:                                                                              │
│   1076 │   │   │   try:                                                                          │
│   1077 │   │   │   │   with self.make_context(prog_name, args, **extra) as ctx:                  │
│ ❱ 1078 │   │   │   │   │   rv = self.invoke(ctx)                                                 │
│   1079 │   │   │   │   │   if not standalone_mode:                                               │
│   1080 │   │   │   │   │   │   return rv                                                         │
│   1081 │   │   │   │   │   # it's not safe to `ctx.exit(rv)` here!                               │
│                                                                                                  │
│ /opt/conda/lib/python3.11/site-packages/click/core.py:1688 in invoke                             │
│                                                                                                  │
│   1685 │   │   │   │   super().invoke(ctx)                                                       │
│   1686 │   │   │   │   sub_ctx = cmd.make_context(cmd_name, args, parent=ctx)                    │
│   1687 │   │   │   │   with sub_ctx:                                                             │
│ ❱ 1688 │   │   │   │   │   return _process_result(sub_ctx.command.invoke(sub_ctx))               │
│   1689 │   │                                                                                     │
│   1690 │   │   # In chain mode we create the contexts step by step, but after the                │
│   1691 │   │   # base command has been invoked.  Because at that point we do not                 │
│                                                                                                  │
│ /opt/conda/lib/python3.11/site-packages/click/core.py:1434 in invoke                             │
│                                                                                                  │
│   1431 │   │   │   echo(style(message, fg="red"), err=True)                                      │
│   1432 │   │                                                                                     │
│   1433 │   │   if self.callback is not None:                                                     │
│ ❱ 1434 │   │   │   return ctx.invoke(self.callback, **ctx.params)                                │
│   1435 │                                                                                         │
│   1436 │   def shell_complete(self, ctx: Context, incomplete: str) -> t.List["CompletionItem"]:  │
│   1437 │   │   """Return a list of completions for the incomplete value. Looks                   │
│                                                                                                  │
│ /opt/conda/lib/python3.11/site-packages/click/core.py:783 in invoke                              │
│                                                                                                  │
│    780 │   │                                                                                     │
│    781 │   │   with augment_usage_errors(__self):                                                │
│    782 │   │   │   with ctx:                                                                     │
│ ❱  783 │   │   │   │   return __callback(*args, **kwargs)                                        │
│    784 │                                                                                         │
│    785 │   def forward(                                                                          │
│    786 │   │   __self, __cmd: "Command", *args: t.Any, **kwargs: t.Any  # noqa: B902             │
│                                                                                                  │
│ /opt/conda/lib/python3.11/site-packages/click/decorators.py:33 in new_func                       │
│                                                                                                  │
│    30 │   """                                                                                    │
│    31 │                                                                                          │
│    32 │   def new_func(*args: "P.args", **kwargs: "P.kwargs") -> "R":                            │
│ ❱  33 │   │   return f(get_current_context(), *args, **kwargs)                                   │
│    34 │                                                                                          │
│    35 │   return update_wrapper(new_func, f)                                                     │
│    36                                                                                            │
│                                                                                                  │
│ /workspaces/tools/nf_core/__main__.py:468 in lint                                                │
│                                                                                                  │
│    465 │                                                                                         │
│    466 │   # Run the lint tests!                                                                 │
│    467 │   try:                                                                                  │
│ ❱  468 │   │   lint_obj, module_lint_obj = run_linting(                                          │
│    469 │   │   │   dir,                                                                          │
│    470 │   │   │   release,                                                                      │
│    471 │   │   │   fix,                                                                          │
│                                                                                                  │
│ /workspaces/tools/nf_core/lint/__init__.py:132 in run_linting                                    │
│                                                                                                  │
│   129 │   # Save results to Markdown file                                                        │
│   130 │   if md_fn is not None:                                                                  │
│   131 │   │   log.info(f"Writing lint results to {md_fn}")                                       │
│ ❱ 132 │   │   markdown = lint_obj._get_results_md()                                              │
│   133 │   │   with open(md_fn, "w") as fh:                                                       │
│   134 │   │   │   fh.write(markdown)                                                             │
│   135                                                                                            │
│                                                                                                  │
│ /workspaces/tools/nf_core/lint/__init__.py:472 in _get_results_md                                │
│                                                                                                  │
│   469 │   │   │   test_failure_count = f"\n-| ❌ {len(self.failed):3d} tests failed       |-"    │
│   470 │   │   │   test_failures = "### :x: Test failures:\n\n{}\n\n".format(                     │
│   471 │   │   │   │   "\n".join(                                                                 │
│ ❱ 472 │   │   │   │   │   [                                                                      │
│   473 │   │   │   │   │   │   f"* [{eid}](https://nf-co.re/tools/docs/{tools_version}/pipeline   │
│   474 │   │   │   │   │   │   f"{strip_ansi_codes(msg, '`')}"                                    │
│   475 │   │   │   │   │   │   for eid, msg in self.failed                                        │
│                                                                                                  │
│ /workspaces/tools/nf_core/lint/__init__.py:473 in <listcomp>                                     │
│                                                                                                  │
│   470 │   │   │   test_failures = "### :x: Test failures:\n\n{}\n\n".format(                     │
│   471 │   │   │   │   "\n".join(                                                                 │
│   472 │   │   │   │   │   [                                                                      │
│ ❱ 473 │   │   │   │   │   │   f"* [{eid}](https://nf-co.re/tools/docs/{tools_version}/pipeline   │
│   474 │   │   │   │   │   │   f"{strip_ansi_codes(msg, '`')}"                                    │
│   475 │   │   │   │   │   │   for eid, msg in self.failed                                        │
│   476 │   │   │   │   │   ]                                                                      │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
NameError: name 'tools_version' is not defined
```

File: nf_core/lint/__init__.py [hunk 1]
```diff
@@ -470,7 +470,7 @@ def _get_results_md(self):
             test_failures = "### :x: Test failures:\n\n{}\n\n".format(
                 "\n".join(
                     [
-                        f"* [{eid}](https://nf-co.re/tools-docs/lint_tests/{eid}.html) - "
+                        f"* [{eid}](https://nf-co.re/tools/docs/{tools_version}/pipeline_lint_tests/{eid}.html) - "
```
Comment (mirpedrol) [inline nf_core/lint/__init__.py:473]: Just saw in the tests that `tools_version` is not defined, we should define it here as in [`format_result()`](https://github.com/nf-core/tools/blob/master/nf_core/lint/__init__.py#L365-L376)

**Comment by mashehu** [comment]
> Docs for v2.10 should exist, maybe there was a problem building them during the release, although everything looks good from the [GitHub action](https://github.com/nf-core/tools/actions/runs/6300992423/job/17105031115).

Looked on the server, the are there, but only in `/latest/` I can try to manually copy them into 2.10

**Comment by mirpedrol** [comment]
@mashehu is this ready to merge?

**Review by mashehu — COMMENTED** [review]
yes, looks good

**Pull request merged**